src/lib.rs

```diff
@@ -496,3 +496,29 @@ impl<'ui> Ui<'ui> {
     pub fn menu<'p>(&self, label: ImStr<'p>) -> Menu<'ui, 'p> { Menu::new(label) }
     pub fn menu_item<'p>(&self, label: ImStr<'p>) -> MenuItem<'ui, 'p> { MenuItem::new(label) }
 }
+
+//Widgets: Combos 
+impl<'ui> Ui<'ui> {
+    pub fn combo<'p>(&self, label : ImStr<'p>, current_item : &mut i32, items : &mut Vec<ImStr<'p>>, items_count : i32, height_in_items : i32) -> bool{
```

Comment:
Can't we just have one function that takes a slice? I don't see any value in having more than one function:

```rust
pub fn combo<'p>(&self, label: ImStr<'p>, current_item: &mut i32, items: &'p [ImStr<'p>], height_in_items: i32) -> bool { }
```